lib/Doctrine/DBAL/Connection.php

```diff
@@ -1151,7 +1151,6 @@ public function lastInsertId($seqName = null)
      *
      * @return mixed The value returned by $func
      *
-     * @throws Exception
      * @throws Throwable
      */
     public function transactional(Closure $func)
```

Comment:
In the light of dropping PHP 5 related code, should this be changed to `callable` instead of `Closure`?

Reply:
Doesn't hurt, but `callable` was available since PHP 5.4!

Reply:
Let's move that to a separate PR if needed :+1: